app/src/main/res/values/colors.xml

```diff
@@ -7,11 +7,16 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="monday">#4BB1F8</color>
+    <color name="tuesday">#53DB89</color>
+    <color name="wednesday">#f98a4b</color>
+    <color name="thursday">#FF5E5E</color>
+    <color name="friday">#838FA4</color>
+    <color name="saturday">#634BFA</color>
```

Comment:
follow something like:
color_{weekday} instead of only {weekday} for naming weekday colors